src/components/presentational/Grid/Grid.stories.tsx

```
@@ -106,4 +107,56 @@ let dailyDataGoals: GridProps = {
 export const DailyDataGoals = {
     args: dailyDataGoals,
     render: render
+};
+
+let trendGrid: GridProps = {
+    children: [
+        <>
+            <Grid.Column span={6}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)